.github/workflows/rust.yml

```diff
@@ -43,18 +43,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-18.04, windows-2019, macos-10.15 ]
+        os: [ubuntu-18.04, windows-2019, macos-10.15]
     steps:
-    # actions/checkout@v2
-    - uses: actions/checkout@230611dbd0eb52da1e1f4f7bc8bb0c3a339fc8b7
-    - uses: actions-rs/toolchain@88dc2356392166efad76775c878094f4e83ff746
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
-    - name: Report cargo version
-      run: cargo --version
-    - name: Build
-      run: cargo build --locked --all-targets --verbose
-    - name: Run tests
-      run: cargo test --locked --verbose
+      # actions/checkout@v2
+      - uses: actions/checkout@230611dbd0eb52da1e1f4f7bc8bb0c3a339fc8b7
+      # Temporary fork until https://github.com/actions-rs/toolchain/pull/209 is merged
```

Comment:
Maybe useful: [RFD 211](https://github.com/oxidecomputer/rfd/tree/master/rfd/0211).

Reply:
This initial version was mostly just to verify the PR via a proper integration. My hope is that the upstream changes will be accepted and we can just stay on the latest release. I can make it an official fork for now though, just so we can move forward without being blocked by the upstream changes.

Reply:
Updated to follow rfd 211 now.